plugins/inputs/snmp/snmp.go

```diff
@@ -566,6 +566,8 @@ type snmpConnection interface {
 	//BulkWalkAll(string) ([]gosnmp.SnmpPDU, error)
 	Walk(string, gosnmp.WalkFunc) error
 	Get(oids []string) (*gosnmp.SnmpPacket, error)
+
```

Comment:
Please remove the newline here.

Reply:
It was to separate it from the Walk and Get methods, but will do if you want it to be like that..